client/helpers/index.ts

```diff
@@ -1 +1,2 @@
 export * from './render-elements.helpers';
+export * from './review-questions.helpers';
```

Comment:
Maybe it makes sense to keep it in `constants`?

Reply:
I don't think functions in a constants file makes sense unless you just mean the `consts` ?

Reply:
Sorry, I meant questions not functions:
```
const S3_YES_NO_QUESTIONS = [
  's3WillATNPSupportActiveTransportationCommuting',
  's3WillTheATNPIncludeAccessToMajorDestinations',
  's3WillTheActiveTransportationNetworkPlanAddressConflictsCausedByTrafficCongestion',
];

const S3_CHECKBOX_VALUES = 's3WillTheActiveTransportationNetworkLinkToAnyOfTheFollowing';
```